src/pymordemos/phlti.py

```diff
@@ -120,27 +121,27 @@ def msd(n=6, m_i=4, k_i=4, c_i=1, as_lti=False):
 
 
 def main(
-        n: int = Option(10, help='Order of the Mass-Spring-Damper system.')
+        n: int = Option(100, help='Order of the Mass-Spring-Damper system.'),
+        r: int = Option(20, help='Order of the reduced model.')
```

Comment:
I guess this causes an issue in the [demo test](https://github.com/pymor/pymor/blob/a641a14ceb6860631524d449d9898655ed52c1a0/src/pymortests/demos.py#L152-L154). Adding the `--r=2` option to the demo test might work.

Reply:
True, I've removed the arguments in the demo test.